[nodejs][03.welcome-users] Corrected sample

joseques opened the pull request:
## Proposed Changes
<!-- Please discuss the changes you have worked on. What do the changes do; why is this PR needed? -->
<!-- You must demonstrate that the code works. Include screenshots of the code in action -->
<!-- If you are introducing a new sample, make sure that the sample adheres to sample guidelines -->
- Replace `.bot` file names in `index.js` to the correct one.
- Rename `.bot` file to match sample name
- Match casing in README.md with bot filename
- Change the section `Deploy this bot to Azure` linking existing documentation
- Change `ConversationState` to `UserState` in `index.js` since this sample has the scope of `UserState`

johnataylor committed:
create folder

carlosscastro committed:
C# DotNetCore update pass for QnaWithAppInsights, LuisWithAppInsights and Dispatch (#507)

* C# DotNetCore update pass for QnaWithAppInsights, LuisWithAppInsights and Dispatch. Renaming directory and moving to samples folder in separate PR

* DotNetCore Dispatch sample documentation fix

* Dotnetcore samples: move QnaAppinsights, dispatch and luisAppinsights to samples directory

* Dotnet samples: folder name casing

daveta committed:
[Final Check] Enterprise Bot asp.net (#511)

* Final Check Enterprise Bot

* Fix some small grammar

JasonSowers committed:
c#/22.conversation-history (#503)

* final pass updates

* final pass updates

* added welcome message

sgellock committed:
Fix bugs package.json and Yeoman generated onTurnError code

- add missing componts key to files section
- add the load, clear, saveChanges pattern to all onTurnError generated code

sgellock committed:
Merge branch 'v4' of https://github.com/Microsoft/BotBuilder-Samples into v4

stevengum committed:
move 17.multilingual-conversations for Node.js and cleanup (#512)

JasonSowers committed:
cleanup

JasonSowers committed:
folder rename

JasonSowers committed:
rename folders

JasonSowers committed:
cleanup

johnataylor committed:
update custom dialogs sample

JasonSowers committed:
cleanup

JasonSowers committed:
cleanup

johnataylor committed:
minor clean up

JasonSowers committed:
Merge pull request #522 from Microsoft/johtaylo/somecleanup

minor clean up

stevengum committed:
Finalize Node.js v4 12.nlp-with-luis sample (#515)

* move 12.nlp-with-luis, fix eslint errors

* fix Node.js v4 sample 12 bot.recipe per @sgellock feedback

stevengum committed:
Finalize Node.js v4 11.qnamaker sample (#514)

* move Node.js 11.qnamaker sample, clean up eslint errors

* fix Node.js 11 bot.recipe per @sgellock feedback

JasonSowers committed:
changed welcome message and minor cleanup

JasonSowers committed:
Merge pull request #520 from Microsoft/johtaylo/19

Johtaylo/19

JasonSowers committed:
clean up c#/03.welcome-user (#513)

* renaming folder

* renaming folder

* namespace cleanup and other fixes

* fixed usings

JasonSowers committed:
readme cleanup

JasonSowers committed:
readme cleanup

stevengum committed:
Finalize Node.js 20.qna-with-appinsights (#524)

* move 20.qna-with-appinsights, fix eslint errors

* upadte bot.recipe for Node.js 20.qna-with-appinsights

daveta committed:
two small fixes (#523)

carlosscastro committed:
Merge pull request #517 from Microsoft/jassow-cleanup-3

Folder rename for #20 and #21

carlosscastro committed:
Merge pull request #521 from Microsoft/jassow-cleanup-5

cleanup c#/06.using-cards

carlosscastro committed:
Merge pull request #518 from Microsoft/jassow-cleanup-4

cleanup c#/05.multi-turn-prompt

carlosscastro committed:
Merge pull request #516 from Microsoft/jassow-cleanup-2

cleanup c#/04.simple-prompt

daveta committed:
Add correct assembly version and fix default.htm, fix Accessor Get to clone

johnataylor committed:
Merge pull request #526 from Microsoft/johndave-echo-webapi

WebAPI Echobot with Counter

johnataylor committed:
folders

JasonSowers committed:
cleanup

stevengum committed:
Finalize Node.js v4 23.luis-with-appinsights (#525)

* move 23.luis-with-appinsights to final place, eslint cleanup

* update .env add eslint

carlosscastro committed:
Merge pull request #527 from Microsoft/jassow-cleanup-5

cleanup c#/07.using-adaptive-cards

Commit:
removing solutions, cafe and skills are in, enterprise in as well.

Commit:
Fixing webchat.

johnataylor committed:
building and updated

daveta committed:
Merge pull request #532 from Microsoft/johtaylo/webapi-projects

Johtaylo/webapi projects

JasonSowers committed:
cleanup and package update (#528)

stevengum committed:
move TS 11.qnamaker, cleanup, bump dependencies (#529)

stevengum committed:
move TS 12.nlp-with-luis, cleanup, add bot.recipe (#530)

Commit:
removing cortana skill - its now under samples folder.

JasonSowers committed:
update all to 4.0.7 (#533)

Commit:
removing work in progress folders.

Commit:
removind dead files.

Commit:
updating all recipe files and deployment scripts

Commit:
updates to readme with nodejs min req for tools

Commit:
luisConfig.getEndpoint() fix

Commit:
adding .lu files to basic template for C#

Commit:
readme updates

Commit:
readme updates #2

Commit:
readme update - fixing / .vs. \

Commit:
reorganizing content

Commit:
removing v3 emulator link

Commit:
merging in jason's PR

vishwacsena committed:
Fixing broken link to v3 samples

enzocano committed:
Add Enterprise Bot Template

Squashed commit:

[37dacb50] Fix typo

[f3ac88d9] Review documentation

[3dc4545e] Add (c) header

[5a377e62] Delete unused classes

[74de8edf] Formatting code

[85835639] Build before start bot

[6d61c025] Change string keys to env.vars

[01a77427] Add bot section to bot.recipe for successful msbot clone services command

[f3ba106c] Add env.development vars

[a35f20c7] Fix LUIS appName

[48464a53] Fix issue with LUIS top intent

[e18ddd86] Migrate documentation and apply minor fixes

[c6677bb0] Various fixes

[1b42ae5a] Padding tsconfig

[dcd46bb7]  Fix resources parsing

[9d8c6f52] Fix propAccessor initialization

[35752d78] Add .map files for easy debug

[11970588] Use single comma

[67fb82a4] Fix resources name

[8c832528] Delete unused resources

[7a8c2807] Add clean and copy-resources scripts

[af5b051a] Implement onboardingDialog.ts and onboardingResponses.ts

[2fdf37e9] Enable use of 'CancelResponses' in 'enterpriseDialog'

[59c40c5f] Fix wrong parameter in 'signInResponses'

[6ca779a8] implemented escalateDialog.ts

[0b06789e] implemented escalateState.ts

[b27c8608] Fix MainDialog properties

[e19c4001] implemented escalateResponses.ts

[65c8994a] implemented cancelState.ts + minor typo fix

[45d7df45] Implement EnterpriseBotAccessors

[04ff4245] Implement cancelResponses.ts

[ed48b4c0] Implement GraphClient

[1bbf84fd] Add SignInDialog

[711d5715] Reorganize imports and remove unused imports

[090d842d] Fix mainDialog

[2acdb8ef] Add files for 'dialogs/shared/'
Add the following files to 'dialogs/shared/':
- interruptableStatus.ts
- interruptableDialog.ts
- enterpriseDialog.ts

[34b2d2ed] Add auto-generated code from LuisGen

[67d6cfbd] Implement mainResponses

[cb07ec2d] Add ActivityEx.createReply

[6308cb4c] Add resourceParser

[9fdc1e7f] Add enterpriseBot

[c3bce667] Fix telemetry send messages

[0cac042d] Add contentModerator middleware

[74a27984] Add luis and qna telemetry

[0cc5ec80] Add TelemetryLoggerMiddleware implementation

[cbe8ccac] Rebase with master

sbuffkin committed:
fix incorrect isSubtle field

JasonSowers committed:
Merge pull request #537 from Microsoft/welcomecardfix

fix incorrect isSubtle field value

Commit:
startup fixes

JasonSowers committed:
Merge pull request #538 from Microsoft/jassow/09

startup fixes for routing bot

daveta committed:
Fix message when bot configuration cannot load

daveta committed:
Merge pull request #539 from Microsoft/daveta-andrea-msg

Fix message when bot configuration cannot load

Commit:
Update Startup.cs

Commit:
Update Startup.cs

daveta committed:
Merge pull request #542 from Microsoft/Andrea-Orimoto-patch-1

C# templates reference a node .env file instead of appsettings.json

sbuffkin committed:
Create .env file for local debugging for custom dialogs bot sample

JasonSowers committed:
Merge pull request #543 from Microsoft/customdialogfix

Create .env file for local debugging for custom dialogs bot sample

stevengum committed:
v4 Node.js samples now point at registry.npmjs.org instead of MyGet, and use 4.0.6

vishwacsena committed:
Fixing minor typo with ./DeploymentScripts/DEPLOY.MD

vishwacsena committed:
Update README.md

johnataylor committed:
Merge pull request #549 from Microsoft/stgum/4.0.6

v4 Node.js samples now use registry.npmjs.org instead of MyGet, and use 4.0.6

vishwacsena committed:
Fix for chitchat .lu files and readme updates. (#548)

Commit:
update csproj to always copy any .bot file when publishing

daveta committed:
Merge pull request #555 from Microsoft/yk-fix-csproj-bot

update csproj to always copy any .bot file when publishing

daveta committed:
Hai msg hygiene

daveta committed:
more hygiene

daveta committed:
Merge pull request #559 from Microsoft/andreao-msghygiene

AndreaO fix: Use verbatim (@) on long msg

kyleburnsdev committed:
Updated cast operations to use pattern matching (#553)

johnataylor committed:
introduce vs solution and clean up project dependencies (#558)

johnataylor committed:
solution and package consolidation in core (#561)

Commit:
update license information (#565)

update content of license file
rename license file to have a markdown extension
update readme to link to license file

Commit:
Fix console-echo to pull from npm, update typescript samples to use 4.0.6 (#570)

the typescript console-echo wasn’t pulling botbuilder package from npm.  fixed this.
also updated all ts samples to use 4.0.6 sdk packages.

Commit:
Fixes #560: Add Typescript prereq for Yeoman generated TypeScript templates (#569)

Add prerequisite section in README.md to TypeScript generated yeoman templates
Add prerequisite section in REAME.md’s of all TypeScript samples
Update yeoman generate’s package.json to have a lisence and repository entries
Update deployment markdown files to use proper msbot clone services syntax
Update yeoman generated package.json files to use 4.0.6 or above of JS SDK

JasonSowers committed:
Update broken link in readme

JasonSowers committed:
Update README.md

JasonSowers committed:
fix broken link in qna samples

JasonSowers committed:
Update README.md

JasonSowers committed:
Update README.md

stevengum committed:
Merge pull request #573 from Microsoft/JasonSower-qna-readme-fix

Update broken link in readme

stevengum committed:
Merge pull request #575 from Microsoft/JasonSowers-link2

fix broken link in qna samples

daveta committed:
Address feedback

vishwacsena committed:
Fix for #436 - VSIX sources. (#572)

daveta committed:
Merge pull request #577 from Microsoft/daveta-cafebot3

Cafebot ASP.Net Core

Commit:
adding sample

Commit:
adding auth bot

vishwacsena committed:
Fix for #436 in samples. (#571)

daveta committed:
Fix tons of issues in here. (#576)

JasonSowers committed:
Fix node/12.nlp-with-luis (#581)

* fixed the bot

* Update README.MD

JasonSowers committed:
Correct package.json

stevengum committed:
change '\' to '/' clean up paths and other errors in .netcore READMEs (#592)

* change '\' to '/' clean up paths and other errors in .netcore READMEs

* additional cleanup on csharp_dotnetcore READMEs

* update .netcore samples 09, 13 READMEs

stevengum committed:
Replace "\" with "/" in JS READMEs and cleanup es6 READMEs (#588)

* update nodejs samples' READMEs

* update ts 02 sample for unix-compat instructions

* cleanup js_es6 READMEs

JasonSowers committed:
Update readme for c#/12.nlp-with-luis (#583)

* Update README.md

* Update README.md

* Update README.md

stevengum committed:
add Installing_CLI_tools.md (#587)

vishwacsena committed:
adding link to c# cafe bot (#589)

JasonSowers committed:
Edit readme for node/12.nlp-with-luis (#584)

Making the same as the C# PR #583

Commit:
Connection setting variable was undefined when signing out (#591)

benbrown committed:
Add eslint configuration

daveta committed:
Fix missing default.htm and accomodate any underscores in bot name for LUIS/Dispatch

JasonSowers committed:
Merge pull request #582 from Microsoft/jassow/18.bot-auth

Jassow/18.bot auth

worzy committed:
Fix filename in logged output when running nodejs sample 14 (#597)

benbrown committed:
Correct formatting

JasonSowers committed:
Merge pull request #599 from Microsoft/benbrown-patch-1

Correct formatting

JasonSowers committed:
Merge pull request #598 from Microsoft/daveta-dispatch2

Fix missing default.htm and accommodate any underscores in bot name

enzocano committed:
Update main readme

enzocano committed:
Fix tslint issues

Commit:
Fix random number generation sample 7

Commit:
[Yeoman] TypeScript Basic template prompt bug (#603)

* add package.version in various places

- Display package version when the generator is run
- Write the package version out in the generated package.json files
- bump the version number of the generator

* Fixes 585 - [Yeoman] TypeScript Basic template prompt bug

- TypeScript generated code was missing a break statement.
- Also updated the inline comments to be consistent with the JavaScript version
- fix yeoman generated code
- fix JavaScript sample
- fix TypeScript sample

Commit:
Fixes 556 - Subscription key in C# #14 readme #556 (#604)

- remove subscription key
- fix markdown formatting

enzocano committed:
Fix DEPLOYMENT.md link

JasonSowers committed:
Merge pull request #602 from Microsoft/kydela

Fix random number generation sample 7

enzocano committed:
Change ContentModerator config key snippet to match bot.recipe

enzocano committed:
Check QnA service name

stevengum committed:
update .netcore Dispatch sample to get into a working state

stevengum committed:
merge master into stgum/csharp_dispatch

jcscottiii committed:
Remove repeating word in README

This commit cleans up the README by removing the repeating, successive words

enzocano committed:
Delete unused code

VictorGrycuk committed:
Add update step for VS Code, improve format

joseques committed:
Fixed minor typos for README

stevengum committed:
Merge pull request #614 from jcscottiii/patch-1

Remove repeating word in README

stevengum committed:
fix v4 Node.js graph sample

JasonSowers committed:
Merge pull request #621 from Microsoft/stgum/node-graph-fix

fix v4 Node.js graph sample

dfavretto committed:
Readme fixes (#610)

alermant committed:
Fix broken links (#617)

daveta committed:
Fix readme commands for msbot clone (#586)

* Fix readme commands for msbot clone

* Fixup more references to js

* Use fully designated forms of command line parameter so more self describing

JasonSowers committed:
update c#/24.bot-authentication-msgraph (#612)

* fixed sample

* cleanup

lauren-mills committed:
Merge pull request #608 from southworkscom/southworks/netcore/fix-52.enterprise-bot

[netcore][52.enterprise bot] Fix ContentModerator service key and QnA service name

lauren-mills committed:
Merge pull request #615 from southworkscom/southworks/netcore/clean-52.enterprise-bot

[netcore][52.enterprise bot] Delete unused code

mr-yamraj committed:
typo those to that

JasonSowers committed:
Merge pull request #626 from mr-yamraj/master

typo those to that

Commit:
fix: fix unreachable code and potential uncaught exception (#632)

talon2king committed:
Fixes to CafeBots OnTurnProperty and BookTableDialog classes

enzocano committed:
FIx typos

VictorGrycuk committed:
Rephrase 'Hit F5'

enzocano committed:
Fix typos

enzocano committed:
Use nodejs slash

stevengum committed:
Merge pull request #642 from southworkscom/southworks/netcore/console-echo-readme

[netcore][01.console echo] Add missing step and improve format in Readme

stevengum committed:
Merge pull request #641 from southworkscom/southworks/nodejs/typo-15.handling-attachments

[nodejs][15.handling-attachments] Fix typos

stevengum committed:
Merge pull request #640 from southworkscom/southworks/typescript/console-echo

[typescript][01.console echo] Fix minor typos for README

stevengum committed:
Merge pull request #638 from southworkscom/southworks/nodejs/typo-14.nlp-with-dispatch

[nodejs][14.nlp-with-dispatch] Fix typos

talon2king committed:
Cleaned up code and removed comments, todos and old code

luisprieto committed:
remove unused requiring modules in nodejs message-routing sample

enzocano committed:
Add proactive message concept to readme

enzocano committed:
Add "show" reference option

joseques committed:
Fix typos and changes in README

joseques committed:
Fixed wrong .bot filenames

enzocano committed:
Change .bot reference

VictorGrycuk reviewed (changes requested):
For [Deploy this bot to Azure](https://github.com/southworkscom/BotBuilder-Samples/blob/088536f34efba724bca8f8d70b25f9d8d01ea267/samples/javascript_nodejs/03.welcome-users/README.MD#deploy-this-bot-to-azure), see if  you can replicate the way it is explained [here](https://github.com/southworkscom/BotBuilder-Samples/blob/00d5391cd660b93c35cd9ff1c0211463e7d7b29d/samples/csharp_dotnetcore/17.multilingual-bot/readme.md#deploy-this-bot-to-azure).

Commit:
Replaced deploy to azure instruction

joseques committed:
Changed ConversationState to UserState

stevengum committed:
Merge pull request #650 from southworkscom/southworks/nodejs/fix-17.multilingual-conversations

[nodejs][17.multilingual-conversations] Edit .bot reference

stevengum committed:
Merge pull request #647 from southworkscom/southworks/nodejs/echo-w-counter

[nodejs][02.echo with counter] Fixed minor typos for README

stevengum committed:
Merge pull request #646 from southworkscom/southworks/nodejs/fix-16.proactive-messages

[nodejs][16.proactive-messages] Add documentation about proactive messages

stevengum committed:
Merge pull request #644 from luisprieto/master

remove unused requiring modules in nodejs message-routing sample

Commit:
resized images to work on all channels

daveta committed:
Merge pull request #652 from Microsoft/jassow-image

c#/node Attachments sample resized images to work on channels

lauren-mills committed:
Merge pull request #601 from southworkscom/southworks/typescript/enterprise

[TypeScript] Enterprise Bot

daveta committed:
Bit updated on readme, pull from appsettings correctly

halilb committed:
fixing watch script for typescript samples

daveta committed:
Relax validation on Echobot, tighten screws on BasicBot

vishwacsena committed:
Cafebot nodejs sample cleanup (#633)

* cleanup cafe bot.

* more fixes. trimming to one luis call per turn.

* more cleanup

* More fixes.

* eslint fix.

* recipe file fixes.

* clean up.

* Updates to cafe bot.

enzocano committed:
[nodejs][19.custom-dialogs] Fix dependencies & readme.md (#651)

* Delete unused dependencies

* Add nodemon run

* Use nodejs-style file names

Commit:
[netcore][11.qnamaker] Fix typos and improve formatting (#656)

* Edit README.md

* Fix typos

* Remove extra space from comment

enzocano committed:
[nodejs][21.luis-with-appinsights] Fix luis configuration & typos (#666)

* Use luisConfig.getEndpoint()

* Use authoringKey if not subscriptionKey

* Fix typos

* Change endpointKey to authoringKey

Commit:
[netcore][07.adaptive cards] Edit Readme (#611)

* Edit readme.md

* Fix typos in code comments

* Remove Azure storage section as per feedback

* Edit deployment step

Commit:
[typescript][02.echo with counter] Fix typos and improve structure in Readme (#663)

* Fixed minor typos for README

* Changed 'Deploy to Azure' section

Commit:
[netcore][02.echo with counter] Fix typos and improve Readme.md (#672)

* Fix typos and update Readme.md

* Apply feedback corrections

Commit:
[netcore][13.basic bot] Fix typos and improve formatting (#671)

* Fix file name reference, deploy section, and style

* Fix typos in code and minor style edits

* Fix typo

Commit:
[netcore][06.using cards] Fix typos and update Readme structure and content (#670)

* Fix typos in comments

* Edit Readme
- Add Testing section
- Improve formatting
- Reorder min req for tool version

* Fixes to Readme.md

Commit:
[typescript][13.basic bot] Fix typo and add missing link in Readme (#667)

* Fixed minor typos for README.md and DEPLOY.md

* Minor typo in bot.ts

nishanc committed:
Update to bot authorization link.

Previous link points to SDK v3 release related article.

VictorGrycuk commented on samples/javascript_nodejs/03.welcome-users/README.MD:
````diff
@@ -38,15 +38,7 @@ Bots that are added directly by a user, are mostly personal (1:1) conversation b
-```bash
-npm i -g msbot chatdown ludown qnamaker luis-apis botdispatch luisgen
-```
+You can use the [MSBot](https://github.com/microsoft/botbuilder-tools) Bot Builder CLI tool to clone and configure any services this sample depends on. In order to install this and other tools, you can read [Installing CLI Tools](../../../Installing_CLI_tools.md).
````
Please see if you can update this section to the latest one used in [netcore](https://github.com/southworkscom/BotBuilder-Samples/blob/78a021e12898d684f8c48bb72c6c2ca332e62c99/samples/csharp_dotnetcore/13.basic-bot/README.md#deploy-this-bot-to-azure).

VictorGrycuk reviewed (changes requested):
For [line 12](https://github.com/southworkscom/BotBuilder-Samples/pull/88/files#diff-8318bbe5906d2e8faa3e10876f459391R12), use `code formating` for the path.

daveta committed:
Merge pull request #676 from nishanc/patch-1

Update to bot authorization link.

Commit:
[netcore][15.handling attachments] Fix typos and improve formatting (#680)

* Edit readme.md

* Fix typos and minor format

* Add missing steps for BFM V4, and deploy

* Fix typos in attachments section

* Fix readme structure

enzocano committed:
Update readme (#679)

enzocano committed:
[nodejs][24.bot-authentication-msgraph] Fix readme.md (#677)

* Order required permissions

* Fix auth documentation link

Commit:
[netcore][09.message routing] Fix typos and text format (#655)

* Fix typos and minor style edits

* Fix typos and broken param name

enzocano committed:
[nodejs][20.qna-with-appinsights] Fix dependencies & readme.md (#654)

* Fix typos

* Change ABS name

* Update .bot filename

Commit:
[netcore][05.multi turn prompt] Fix typos and add improvements (#620)

* Edit readme.md

* Fix typos

* Remove Azure Storage section as per feedback

* Edit deployment step

Commit:
[netcore][03.welcome user] Fix typos and add improvements (#619)

* Edit readme.md

* Fix typos and broken param name

* Remove Azure Storage section as per feedback

* Edit deployment step

JasonSowers committed:
Merge pull request #637 from talon2king/master

Fixes to CafeBots OnTurnProperty and BookTableDialog classes

franher committed:
refactor: ES6 refactors (#678)

* refactor: 'didBotWelcomeUser' is never reassigned. Use 'const' instead

* refactor: use let instead of var

* refactor: remove unused parameter

* refactor: use let instead of var

* refactor: Delete unnecessary return await

* refactor: 'userName' is never reassigned. Use 'const' instead

myselfhimself committed:
Add missing fetch-ponyfill dependency (#648)

Otherwise `npm run start` crashes with `Error: Cannot find module 'fetch-ponyfill'`

daveta committed:
Just fix the readme this time

stevengum committed:
Merge pull request #661 from halilb/master

fixing watch script for typescript samples

JasonSowers committed:
removing bot files for samples with services (#685)

* removed bot files for c#

* removed bot files for c#

* removing bot files

* removing bot files

* removing bot files

JasonSowers committed:
Merge pull request #673 from Microsoft/daveta-publish-validate

Modify echobot and basicbot to add Publish validations

Commit:
[nodejs][01.console echo] Fix typo and add missing ampersand (#694)

* Changed single ampersand to double ampersand

* Typo in consoleAdapter.js file

Commit:
[es6][1.a.browser echo] Fix missing ampersand and udpate botbuilder-core version (#691)

* Updated botbuilder-core version

* Change ampersand in README

lucaleone committed:
Fix Startup.cs typos (#688)

fixed typos in Startup.cs comments

lucaleone committed:
removed rendundant field initializer + readonly + improved LINQ query (#687)

_isProduction has been made readonly to avoid user changes.
The linq query at line 68 has been mad shorter to improver readability.

enzocano committed:
[netcore][20.qna-with-appinsights] Fix readme.md and issue with DI (#689)

* Fix typos

* Fix dependency injection

Commit:
Replaced single ampersand for double ampesand (#700)

Commit:
[typescript][11.qnamaker] Fix typos and improve formatting (#702)

* Fixed minor typos for README

* Fix .qna file invalid JSON format

* Fixed syntax error in .bot file

* Add missing comma in .qna file

enzocano committed:
Add steps to run the sample (#695)

enzocano committed:
[netcore][18.bot-authentication] Fix typos and Add OAuth configuration steps (#693)

* Fix typos

* Add step to configure OAuth

dfavretto reviewed (changes requested):
Fix typo in [line 97](https://github.com/southworkscom/BotBuilder-Samples/blob/d04a523fe3f1dc6504d23016c5b4882bbe6a1d88/samples/javascript_nodejs/03.welcome-users/bot.js#L97) file `bot.js`

stevengum committed:
update the README, .luis, .lu and .qna files

daveta committed:
Merge pull request #684 from Microsoft/daveta-nodejs

Fix nodejs dispatch

stevengum committed:
fix merge conflicts

stevengum committed:
update README.md, NlpDispatchBot.cs, CognitiveModels

stevengum committed:
include appsettings.json instructions in README.md

stevengum committed:
add LUIS subscription key link to README

stevengum committed:
change static readonly to consts in .netcore dispatch sample

Commit:
Fix typos

benbrown committed:
Change ILuisService (abstract interface) to LuisService (implemented real class)
Update readme to add instructions about format .bot file
Add "future reading" link

benbrown committed:
Add a note about missing luis configuration value
Remove placeholder value for luis configuration

benbrown committed:
readme formatting tweaks

benbrown committed:
readme formatting tweaks

stevengum committed:
Merge pull request #713 from Microsoft/benbrown/fix-627

Update nlp-with-luis (fix for #627)

benbrown committed:
Update readme to include instructions to set LUIS_CONFIGURATION
Update readme with some new links to match the typescript version of this
Remove unused blob storage config from bot.recipe
Add missing 34.luis file to deploymentScripts

stevengum committed:
change public consts to private

stevengum committed:
apply fixes to node basic-bot sample per #634

Commit:
fixing welcome messages

Commit:
fixing welcome messages

Commit:
fixing welcome messages

Commit:
renamed context

benbrown committed:
Update samples/javascript_nodejs/13.basic-bot/deploymentScripts/DEPLOY.md

stevengum committed:
Merge pull request #717 from Microsoft/stgum/node-basic

apply fixes to node basic-bot sample per #634

stevengum committed:
Merge pull request #613 from Microsoft/stgum/csharp_dispatch

 update .netcore Dispatch sample

JasonSowers committed:
Merge pull request #718 from Microsoft/jassow-welcome

Update welcome messages

stevengum committed:
update README.md and fix index.js in nlp-with-luis (#630)

stevengum committed:
Merge pull request #715 from Microsoft/benbrown/fix-625

Update pure js version of nlp-with-luis (fix for #625)

stevengum committed:
Merge pull request #720 from Microsoft/stgum/node-luis

update README.md and fix index.js in nlp-with-luis (#630)

Commit:
Corrected routing in bot.js and param assignment in greeting.js

Commit:
Removed duplicated bot.js code

alundiak committed:
Small typo fixes in comments and string value for 'description' field in result 'package.json' file. (#722)

Commit:
[nodejs][03.welcome users] Fix typos and wrong parameter type, and improve formatting (#719)

* Fixed wrong .bot filenames

* Replaced deploy to azure instruction

* Changed ConversationState to UserState

* Fix typos

Commit:
[netcore][19.custom dialogs] Fix typos and improve formatting (#704)

* Fix typos and minor style edits

* Fix typos and minor style edit

* Move static member before non-static member

enzocano committed:
Add steps to setup blob storage (#710)

Commit:
[netcore][17.multilingual bot] Fix typos and improve formatting (#701)

* Minor style edits, edit Deploy section

* Fix typos and minor style changes

* MultiLingualBot.bot

* Fix additional typos

Commit:
[nodejs][04.simple prompt] Fix typos and improve formatting (#725)

* Fix typos and enhance readme structure

* Fix typos in sample scripts

* Added language to code style

JasonSowers committed:
c#/20.qna-with-appinsights (#716)

* added qna

* Update 34.qna

carlosscastro committed:
Message routing nodejs sample: minor fixes and improvements

carlosscastro committed:
Message routing sample fixes: PR comments

Commit:
Updates to reflect PR change requests

stevengum committed:
Merge pull request #727 from Microsoft/ccastro/fix-multilingual-node

Multilingual-conversations Node.js sample: minor fixes and improvements

Commit:
Added descriptor for utterance param

stevengum committed:
Merge pull request #721 from Microsoft/vstkanb-message-routing

vstkanb message routing

lucaleone committed:
Added WelcomeUserState + fix Exception message + improved Linq query (#696)

WelcomeUser project:
Implemented same architecture of the EchoBotWithCounter project. The project now contains a WelcomeUserState class handled by WelcomeUserStateAccessors.

line 62 all Startup.cs
botConfig in the InvalidOperationException message is always null, it would be better for the user to provide the path in which the .bot file shold be.

Line 66 all Startup.cs
Improved LINQ query

Commit:
Missed typos and wrong path for sample folder in README

joseques committed:
Merge branch 'master' into southworks/nodejs/welcome-users
